be/src/storage/rowset_column_update_state.cpp

```diff
@@ -384,11 +377,27 @@ static Status read_chunk_from_update_file(const ChunkIteratorPtr& iter, const Ch
 // 1. get the rows from update files, store in `result_chunk`
 // 2. generate `rowids`, the rowid list marks the rows in source segment file which be updated.
 Status RowsetColumnUpdateState::_read_chunk_from_update(const RowidsToUpdateRowids& rowid_to_update_rowid,
```

Comment:
We have a file descriptor cache named `FdCache`. I think you can place the open file descriptor through the cache. It will auto reclaim and reuse the file descriptor.

Reply:
I don't think we need `FdCache` here. Once `.upt` file is open and read, it's data will be stored in `_update_chunk_cache`, it won't be reused again.